src/uwtools/utils/cli_helpers.py

```diff
@@ -67,3 +68,22 @@ def setup_logging(user_args, log_name=None): #pylint: disable=unused-variable
         log.propagate = False
 
     return log
+
+def compare_files(expected, actual):
```

Comment:
This might not be the best place for this method given it's not really a command line interface helper. Maybe file_helpers.py is a more appropriate location?